docs/gitworkflow.rst

```diff
@@ -157,8 +162,9 @@ merged into the upstream repository:
 
    You will then have to derive a new feature branch from the reset `master`
    branch and then add your changes manually to this new feature
-   branch. Therefore, to avoid this extra work, make sure you never change your
-   personal `master`, apart from synchronising it with the upstream `master`.
+   branch. Therefore, to avoid this extra work, make sure you **never change
+   your personal `master` branch**, apart from synchronising it with upstream
+   `master`.
```

Comment:
Italics should be sufficient for highlighting, I don't think bold typeface is needed here.